core/src/main/java/org/springframework/security/core/SpringSecurityMessageSource.java

```diff
@@ -37,8 +36,4 @@ public SpringSecurityMessageSource() {
 		setBasename("org.springframework.security.messages");
 	}
 
-	public static MessageSourceAccessor getAccessor() {
```

Comment:
This cannot be removed as it breaks compatibility. Spring Security follows [semantic versioning](https://semver.org/).